frontend/src/v5/ui/routes/dashboard/projects/containers/containersList/containersList.helpers.ts

```diff
@@ -15,54 +15,78 @@
-		title: formatMessage({ id: 'containers.ellipsisMenu.viewIssues', defaultMessage: 'View Issues' }),
-		onClick: () => { },
-	},
-	{
-		key: 4,
-		title: formatMessage({ id: 'containers.ellipsisMenu.viewRisks', defaultMessage: 'View Risks' }),
-		onClick: () => { },
-	},
-	{
-		key: 5,
-		title: formatMessage({ id: 'containers.ellipsisMenu.viewRevisions', defaultMessage: 'View Revisions' }),
-		onClick: () => { },
-	},
-	{
-		key: 6,
-		title: formatMessage({ id: 'containers.ellipsisMenu.editPermissions', defaultMessage: 'Edit Permissions' }),
-		onClick: () => { },
-	},
-	{
-		key: 7,
-		title: formatMessage({ id: 'containers.ellipsisMenu.shareContainer', defaultMessage: 'Share Container' }),
-		onClick: () => { },
-	},
-	{
-		key: 8,
-		title: formatMessage({ id: 'containers.ellipsisMenu.settings', defaultMessage: 'Settings' }),
-		onClick: () => {
+export const getContainerMenuItems = (container: IContainer) => {
+	const { teamspace, project } = useParams() as { teamspace: string, project: string };
+	const dispatch = useDispatch();
+	return [
+		{
+			key: 1,
+			title: formatMessage({ id: 'containers.ellipsisMenu.loadContainer', defaultMessage: 'Load Container in 3D Viewer' }),
+			to: `/${container._id}`,
 		},
-	},
-	{
-		key: 9,
-		title: formatMessage({ id: 'containers.ellipsisMenu.delete', defaultMessage: 'Delete' }),
-		onClick: () => { },
-	},
-];
+		{
+			key: 2,
+			title: formatMessage({ id: 'containers.ellipsisMenu.uploadNewRevision', defaultMessage: 'Upload new Revision' }),
+			onClick: () => { },
+		},
+		{
+			key: 3,
+			title: formatMessage({ id: 'containers.ellipsisMenu.viewIssues', defaultMessage: 'View Issues' }),
+			onClick: () => { },
+		},
+		{
+			key: 4,
+			title: formatMessage({ id: 'containers.ellipsisMenu.viewRisks', defaultMessage: 'View Risks' }),
+			onClick: () => { },
+		},
+		{
+			key: 5,
+			title: formatMessage({ id: 'containers.ellipsisMenu.viewRevisions', defaultMessage: 'View Revisions' }),
+			onClick: () => { },
+		},
+		{
+			key: 6,
+			title: formatMessage({ id: 'containers.ellipsisMenu.editPermissions', defaultMessage: 'Edit Permissions' }),
+			onClick: () => { },
+		},
+		{
+			key: 7,
+			title: formatMessage({ id: 'containers.ellipsisMenu.shareContainer', defaultMessage: 'Share Container' }),
+			onClick: () => { },
+		},
+		{
+			key: 8,
+			title: formatMessage({ id: 'containers.ellipsisMenu.settings', defaultMessage: 'Settings' }),
+			onClick: () => {
+			},
+		},
+		{
+			key: 9,
+			title: formatMessage({ id: 'containers.ellipsisMenu.delete', defaultMessage: 'Delete' }),
+			onClick: () => {
+				dispatch(DialogsActions.open('delete', {
+					title: formatMessage(
+						{ id: 'deleteModal.title', defaultMessage: 'Delete {name}?' },
+						{ name: container.name },
+					),
+					onClickConfirm: () => ContainersActionsDispatchers.deleteContainer(
+						teamspace,
+						project,
+						container._id,
+					),
+					message: formatMessage({
+						id: 'deleteModal.message',
```

Comment:
Let's change the id of the formatmessage, otherwise it might get confuse with other potential delete modals (like federations
```suggestion
						id: 'containers.deleteModal.message',
```